romana-install/romana-setup

```diff
@@ -18,12 +18,14 @@
 usage() {
 	echo ""
 	echo "romana-setup: executes the ansible playbooks for installing romana"
-	echo "Usage: romana-setup [-t targetname] [action]"
-	echo "       romana-setup [-t targetname] <action> [ansible-options]"
+	echo "Usage: romana-setup [-n stackname] [-p platform ] [-s stacktype] [action]"
+	echo "       romana-setup [-n stackname] [-p platform ] [-s stacktype] <action> [ansible-options]"
 	echo "       romana-setup <h|--help>"
 	echo ""
-	echo "Targets: aws (default), vagrant"
-	echo "Actions: install (default), uninstall"
+	echo "Stack Name:  used-defined stack name (default: $USER)"
+	echo "Platforms:   aws (default), vagrant"
+	echo "Stack Types: devstack (default)"
+	echo "Actions:     install (default), uninstall"
 	echo ""
 	echo "Example: ./romana-setup install -e stack_name=xyzdemo"
 	echo "Example: ./romana-setup uninstall -e stack_name=xyzdemo"
```

Comment:
Might be nice to show an example with a full set of parameters:

./romana-setup -n mystack -p vagrant -s kubernetes install